src/view-resources.js

```diff
@@ -67,6 +73,72 @@ interface ViewEngineHooks {
 * Represents a collection of resources used during the compilation of a view.
 */
 export class ViewResources {
+
+  static convention(target: Function) {
+    let config = target.resource;
+    let resource;
+    if (config) {
+      if (typeof config === 'string') {
+        // it's a custom element, with name is the resource variable
+        // static resource = 'my-element'
```

Comment:
custom attribute being quite useful (in my opinion), is it worth it to have a shortcut for that as well?
For example:
`"data-grid"` -> custom element named `data-grid`
`"@blink"` -> custom attribute named `blink`

Reply:
the `@` seems nice, but also out of nowhere I think, maybe we can use existing definition: `[custom-attribute]` like css selector ?

Reply:
Yes, that would be more "native" to the web platform. Dunno where I got the `@` idea from. 😀

Reply:
then do we fully embrace symbolic characters ? like `'|valueConverter'` and `'&bindingBehavior'` ? Maybe we can comeback to it later as an enhancement, otherwise it would be quite hard to explain in doc about the inconsistency.

Reply:
Yes, that was an enhancement suggestion. 
We can absolutely come back to it later when we have more feedback on the v1.
`[my-attribute]` is not a valid element name anyway, so it won't be a breaking change to add that kind of support later... if we want to.